src/bioregistry/data/curated_papers.tsv

```diff
@@ -1,7 +1,7 @@
 pmid	relevant	relevancy_type	prefix	notes	pr_added	orcid	date_curated
-39104285	1	new_provider		Provider for UniProt IDs, issue with curation due to multiple URI formats depending on plant species	1193	0009-0009-5240-7463	2024-09-24
-39074139	1	new_provider		Resolver for PDB IDs		0009-0009-5240-7463	2024-09-24
-39014503	0	no_website				0009-0009-5240-7463	2024-09-25
+39104285	1	new_provider	uniprot	Provider for UniProt IDs, issue with curation due to multiple URI formats depending on plant species	1193	0009-0009-5240-7463	2024-09-24
+39074139	1	new_provider	pdb	Resolver for PDB IDs	1193	0009-0009-5240-7463	2024-09-24
```

Comment:
One issue with having an explicit prefix column is that often there will not be a straightforward mapping there. One paper can be broken up into multiple prefixes, on can be a provider for multiple prefixes. Now we could come up with a more general representation like allow a comma separated list of prefixes to be added. But unless really necessary, all these increase the curation burden and make keeping things in sync more difficult.